src/app/page.tsx

```diff
@@ -31,7 +31,7 @@ const HomePage = () => {
                 events in Vancouver, and you can search for events by date.
               </Text>
               <Link href={`/map`}>
-                <Button>Find Events</Button>
+                <Button buttonType="tertiary">Find Events</Button>
```

Comment:
same here. I think you're totally fine with spelling but I'm not...

Reply:
Ah that sounds good! I will make a change!